### DISCUSSION TITLE
how should i test a DAG with CronTriggerTimetable schedule?

### DISCUSSION THREAD

**Discussion by pmak-852** [apache/airflow#34699] [Q&A] [answered]
# referring to Airflow v2.6.3

the DAG below comes from https://airflow.apache.org/docs/apache-airflow/2.6.3/authoring-and-scheduling/timetable.html
``` python3
from airflow.timetables.trigger import CronTriggerTimetable
from airflow.decorators import dag


@dag(schedule=CronTriggerTimetable("0 1 * * 3", timezone="UTC"))  # At 01:00 on Wednesday
def example_dag():
    pass

example_dag()
```
now I want to test the whole dag by calling 
``` python3
example_dag().test()
```
then I receive this error when I run the script
```
ValueError: Not a valid timetable: <airflow.timetables.trigger.CronTriggerTimetable object at 0x000002036A69BD50>
```

Interestingly, this dag can be correctly triggered by airflow scheduler without any exception.

Am I using the **CronTriggerTimetable** wrongly? or i am testing things in the wrong way? 

*PS: this thing works with airflow v2.7.1, not sure if this is a bug in v2.6.x or not*

**Comment by hussein-awala** [comment, marked as answer]
This issue was fixed by #32118 in 2.7.0
